### PR TITLE
Doc for cephfs expand PVC

### PR DESCRIPTION
**Pull request by Madhu-1**
Added Doc on how to expand cephfs PVC and how to  verify it

Signed-off-by: Madhu Rajanna <madhupr007@gmail.com>

### CODE REVIEW

**Comment by humblec** [comment]
@nixpanic @ShyamsundarR can you review/approve this in priority ?

File: README.md [hunk 1]
```diff
@@ -61,21 +61,21 @@ NOTE:
...
 |        | Provision volume from another volume                      | -              | -                  | -                | -                    | -                  |
-|        | Resize volume                                             | -              | -                  | -                | -                    | -                  |
-|        | Metrics Support                                           | -              | -                  | -                | -                    | -                  |
-| CephFS | Dynamically provision, de-provision File mode RWO volume  | Alpha          | >=v1.1.0           | >=v1.0.0         | Nautilus (>=14.2.2)  | >=v1.13.0          |
-|        | Dynamically provision, de-provision File mode RWX volume  | Alpha          | >=v1.1.0           | >=v1.0.0         | Nautilus (>=v14.2.2) | >=v1.13.0          |
+|        | Resize volume                                             | Beta           | >= v2.0.0          | >= v1.1.0        | >= Mimic             | >= v1.15.0         |
```
Comment (nixpanic) [inline README.md]: You mention the version after Nautilus, maybe do so for Mimic too?

Comment (Madhu-1) [reply README.md]: for cephfs version is required,  not for mimic. ceph-csi works with all minor version of Mimic but require 14.2.2 in Nautilus

Comment (nixpanic) [reply README.md]: Ah, now I understand. Can you then just write "Mimic (>= 13.0.0)"? For everything else it also counts as a minimal version, and it is assumed that newer Ceph versions (whatever comes after Nautilus) is also supported.

Comment (Madhu-1) [reply README.md]: done updated now PTAL

**Pull request merged**